src/pocketmine/plugin/PluginManager.php

```diff
@@ -284,6 +284,16 @@ public function loadPlugins(string $directory, array $newLoaders = null){
 						}
 					}
 
+					if($description->getMinPHPVersion() !== ""){
+						if(version_compare($description->getMinPHPVersion(), PHP_VERSION) > 0){
+							$this->server->getLogger()->error($this->server->getLanguage()->translateString("pocketmine.plugin.loadError", [
```

Comment:
It is a better idea to use semver-based version matching, just like how we match API versions. Supporting 7.3.2 should only imply support for 7.3.{≥2} and 7.{≥4}.?, but not 8.?.?.

Reply:
for posterity, recreating the same comment here that I wrote on discord: PHP does not use semver, breaks can happen on minor versions